HackYourCareer/SmartKickers
# Kubernetes Go Deployment files

# Summary of Changes

This PR adds files necessary for deploying our Go server to the Kubernetes cluster. Resolves: #115 

# How to test

1. Clone Repository
2. Setup K8S cluster
3. Apply "deployment.yml"
4. Check if the backend is working correctly (using Postman or Node.js and React)
